### PR TITLE
feat: added dot style to badge

### PR DESCRIPTION
**Pull request by StefanDywersant**
Added dot variant to Badge component

### Screenshot
zrzut ekranu 2019-02-26 o 11 13 58

### CODE REVIEW

File: src/__tests__/Badge.spec.tsx [hunk 1]
```diff
@@ -23,9 +23,24 @@ describe('Badge component', () => {
...
+            padding: '0',
+            width: '8px',
+            height: '8px',
+            display: 'inline-block',
+          },
+        ])
+      );
+    });
+
+    it('do not renders children when in dot variant', () => {
```
Comment (P0lip) [inline src/__tests__/Badge.spec.tsx]:
```suggestion
    it('do not render children when in dot variant', () => {
```

File: src/__tests__/Badge.spec.tsx [hunk 2]
```diff
@@ -23,9 +23,24 @@ describe('Badge component', () => {
...
+      const wrapper = shallow(<Badge variant={BadgeVariant.Dot}>test</Badge>).shallow();
+      expect(wrapper.contains('test')).toBe(false);
```
Comment (P0lip) [inline src/__tests__/Badge.spec.tsx]: Could be
```suggestion
      expect(wrapper).not.toHaveText('test');
```

File: src/__tests__/Badge.spec.tsx (hunk 1 above)
Comment (StefanDywersant) [reply src/__tests__/Badge.spec.tsx]: 👍

File: src/__tests__/Badge.spec.tsx (hunk 2 above)
Comment (StefanDywersant) [reply src/__tests__/Badge.spec.tsx]: 👍

**Pull request merged**